tests/test_examples.py

```diff
@@ -520,6 +520,26 @@ def test(self):
                 env_variables["PT_HPU_LAZY_MODE"] = "0"
                 env_variables["PT_ENABLE_INT64_SUPPORT"] = "1"
 
+            if self.EXAMPLE_NAME == "run_glue":
+                if model_name == "bert-large-uncased-whole-word-masking":
+                    extra_command_line_arguments.append("--sdp_on_bf16")
+
+            if self.EXAMPLE_NAME == "run_qa":
+                if model_name == "bert-large-uncased-whole-word-masking" or \
+                   model_name == "albert-large-v2":
+                    extra_command_line_arguments.append("--sdp_on_bf16")
+
+            if self.EXAMPLE_NAME == "run_bridgetower":
+                if model_name == "BridgeTower/bridgetower-large-itm-mlm-itc":
+                    extra_command_line_arguments.append("--sdp_on_bf16")
+
+            if self.EXAMPLE_NAME == "run_speech_recognition_seq2seq":
+                if model_name == "openai/whisper-small":
```

Comment:
This is not updated in README. should I update?

Reply:
we need to update readme for all these with PR 1514
contrastive-image-text/run_bridgetower.py
question-answering/run_qa.py
speech-recognition/run_speech_recognition_ctc.py
stable-diffusion//text_to_image_generation.py
text-classification/run_glue.py

Reply:
right. those READMEs are all updated. But I see that this model uses run_speech_recognition_seq2seq.py

Reply:
yes we need update that too.